R/theme.R

```diff
@@ -0,0 +1,64 @@
+#' A ggplot2 theme for IHEID style plots
+#' 
+#' @importFrom ggplot2 theme
+#' @examples 
+#' \dontrun{
+#' ggplot(iris, aes(x = Sepal.Length, y = Sepal.Width)) + 
+#' geom_point() + ggtitle("Example Plot") + theme_iheid()
+#' }
+#' @export
+theme_iheid <- function(){
+  font <- "Helvetica"
+  
+  ggplot2::theme(
+    
+    #Text format:
+    #This sets the font, size, type and colour of text for the chart's title
+    plot.title = ggplot2::element_text(family=font,
```

Comment:
Need to declare these additional ggplot2 @importFrom?

Reply:
Not sure I understood you here. Could you rephrase the comment?

Reply:
Quicker to just do it. Done in last commit.

Reply:
Thanks, just saw it and read the Roxygen doc. Everything is much clearer now.